conda/environment-test-3.10.yml

```diff
@@ -1,14 +1,22 @@
-name: cunumeric-test
+name: legate-core-test
 channels:
   - conda-forge
-  - defaults
 dependencies:
   - python=3.10
 
   # build
-  - cutensor>=1.3.3
+  - git
   - nccl
+  - make
+  - zlib
+  - cmake>=3.24
+  - ninja
+  - c-compiler
+  - cxx-compiler
+  - gcc_linux-64 # [linux64]
+  - sysroot_linux-64==2.17 # [linux64]
```

Comment:
Are we doing something to remove the `-O?` (and other) flags that conda-forge's compiler package injects into `CXXFLAGS` and friends? See https://github.com/conda-forge/ctng-compiler-activation-feedstock/issues/80

If we're not, then these compilers don't allow us to do debug builds, and thus are not useful for development.

Reply:
CMake will append [our flags](https://github.com/jjwilke/legate.core/blob/add-cmake-pr/CMakeLists.txt#L70-L80) after the ones specified by conda, and ours will take precedence. There may be extra flags we need to remove, though I haven't had any issues so far.